Source/CarthageKit/Project.swift

```diff
@@ -319,11 +319,17 @@ public final class Project {
 	///
 	/// This will fetch dependency repositories as necessary, but will not check
 	/// them out into the project's working directory.
-	public func updatedResolvedCartfile() -> SignalProducer<ResolvedCartfile, CarthageError> {
+	public func updatedResolvedCartfile(targetDependencies: [String] = []) -> SignalProducer<ResolvedCartfile, CarthageError> {
```

Comment:
It'd be more accurate to represent this as `dependenciesToUpdate: [String]? = nil`. (The type being the important bit; the name is a suggestion.)